.github/ISSUE_TEMPLATE/propose-a-talk.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Propose a talk
+about: Propose a talk or workshop for upcoming meetup/event
+title: ''
+labels: ''
```

Comment:
How about we define a tag for proposals? something like `talk-proposal`

Reply:
Yes, we should have that label, `talk-proposal`

Reply:
I have renamed the existing `new talk` label to `talk-proposal`

Reply:
Can we also include level: Beginner, Intermediate, Advance?

Reply:
@chkumar246 I think we should keep it simple for now, we can always reiterate over the template later